modin/pandas/test/test_io.py

```diff
@@ -863,7 +903,10 @@ def test_read_csv_s3(self):
     @pytest.mark.parametrize("names", [list("XYZ"), None])
     @pytest.mark.parametrize("skiprows", [1, 2, 3, 4, None])
     def test_read_csv_skiprows_names(self, names, skiprows):
-
+        if Backend.get() == "Omnisci" and names is None and skiprows in [1, None]:
```

Comment:
did you forget to add `not` to `skiprows` check?

Reply:
Not sure we should add `not` here, this test marked as `xfailed` because if one row will be skipped or rows won't be skipped at all, columns names will be inferred from the first row, that will contain duplicated values, that is not supported by  `Omnisci` backend yet. Corresponding comment is added.